plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -244,6 +245,7 @@
     protected long _diskActivityCheckFileSizeMin = 10485760; // 10MB
     protected int _diskActivityCheckTimeoutSeconds = 120; // 120s
     protected long _diskActivityInactiveThresholdMilliseconds = 30000; // 30s
+    protected String _qemuSocketsPath;
```

Comment:
Why not represent this value as a ``File`` instead of a ``String``?  The ``File`` class provides semantics for path construction, as well as, providing stronger type information.  Also, why is it declared ``protected`` and not ``private``?